Source/WebCore/platform/graphics/mac/controls/InnerSpinButtonMac.mm

```diff
@@ -43,13 +44,25 @@
 
 WTF_MAKE_TZONE_ALLOCATED_IMPL(InnerSpinButtonMac);
 
-InnerSpinButtonMac::InnerSpinButtonMac(InnerSpinButtonPart& owningPart, ControlFactoryMac& controlFactory)
+InnerSpinButtonMac::InnerSpinButtonMac(InnerSpinButtonPart& owningPart, ControlFactoryMac& controlFactory, NSStepperCell* stepperCell)
     : ControlMac(owningPart, controlFactory)
+    , m_stepperCell(stepperCell)
 {
 }
 
 IntSize InnerSpinButtonMac::cellSize(NSControlSize controlSize, const ControlStyle&) const
 {
+#if HAVE(NSSTEPPERCELL_INCREMENTING)
+    if ([m_stepperCell respondsToSelector:@selector(setIncrementing:)]) {
+        static const IntSize sizes[] = {
+            { 19, 28 },
+            { 15, 22 },
+            { 13, 18 },
+            { 19, 28 }
```

Comment:
I see these metrics are slightly different from the existing. We also need to update `stepperSizes()` in `ThemeMac`.

We really need to have a single source of truth, and then also make these programmatically determined, but not asking for that in this patch 🙂